websites/A/AniWorld/dist/metadata.json

```
@@ -4,23 +4,24 @@
 		"name": "Emijor",
 		"id": "466885774621540372"
 	},
-	"service": "AniCloud",
+	"service": "AniWorld",
 	"description": {
-		"en": "Watch now over 1,000+ animes for free on AniCloud. Anime Video-On Demand: ✓ 100% Free ✓ Instantly Online ✓ Exclusive Animes",
-		"de": "Jetzt über 1.000+ Animes als Stream kostenlos bei AniCloud ansehen. Anime Video-On-Demand: ✓ 100% Kostenlos ✓ Sofort online ✓ Exclusive Animes",
-		"nl": "Bekijk nu meer dan 1.000+ animes gratis op AniCloud. Anime Video-On Demand: 100% Gratis ✓ Direct Online ✓ Exclusieve Animes",
-		"vi_VN": "Xem ngay hơn 1,000+ bộ anime tại AniCloud hoàn toàn miễn phí. Anime theo nhu cầu: ✓ 100% miễn phí ✓ Trực tuyến ✓ Những bộ anime độc quyền"
+		"en": "Watch now over 1,000+ animes for free on AniWorld. Anime Video-On Demand: ✓ 100% Free ✓ Instantly Online ✓ Exclusive Animes",
+		"de": "Jetzt über 1.000+ Animes als Stream kostenlos bei AniWorld ansehen. Anime Video-On-Demand: ✓ 100% Kostenlos ✓ Sofort online ✓ Exclusive Animes",
+		"nl": "Bekijk nu meer dan 1.000+ animes gratis op AniWorld. Anime Video-On Demand: 100% Gratis ✓ Direct Online ✓ Exclusieve Animes",
+		"vi_VN": "Xem ngay hơn 1,000+ bộ anime tại AniWorld hoàn toàn miễn phí. Anime theo nhu cầu: ✓ 100% miễn phí ✓ Trực tuyến ✓ Những bộ anime độc quyền"
 	},
-	"url": "anicloud.io",
-	"version": "1.0.13",
+	"url": "aniworld.to",
+	"version": "1.2.0",
```

I know I mentioned in the thread that it should be bumped, but as you renamed the presence folder it is theoretically a new presence
```suggestion
	"version": "1.0.0",
```